Assets/Source/Options.cs

```diff
@@ -4,13 +4,36 @@
 namespace HiveMind
 {
 	public class Options : MonoBehaviour {
```

Comment:
@rmjohnson Ok. The basic motivation here is to get the Options class into the editor.

Static class properties do not show up in the Unity editor because they are not tied to any specific object. So I changed all of the options to member variables.

Of course, then we have to have a reference to the Options object from anywhere in the code, so basically, I decided to create a singleton. When an Options object is created by the Unity editor by being bound to an object in the scene, a reference to that object is stored in a static variable `private static Options options;`. You can then refer to this instance anywhere in the code using syntax similar to `Options.O.member`.

This isn't fool proof design as adding multiple Options objects to a scene may cause havok, but it works as long as we are careful. I'd be open to a better way of doing this that isn't tied to a behavior but still allows altering from the Unity editor. Maybe a custom project settings thing is possible.

Let me know if that explains the changes sufficiently. It made sense to me when I did it and it still seems like an ok change to me at the moment.